tsdb/exprs.go

```diff
@@ -0,0 +1,90 @@
+package tsdb
```

Comment:
Weren't some of the functions in this file merged into the influxql repo? Do we still need this new file in influxdb?